doc/api/stream.md

```diff
@@ -1725,6 +1725,49 @@ async function showBoth() {
 showBoth();
 ```
 
+### `readable.map(fn[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+* `fn` {Function|AsyncFunction} a function to map over every item in the stream.
```

Comment:
If I understand the code correctly, `fn` receives two parameters, the value from the stream and an object which contains an abort signal, but I don't see it mentioned in the docs.

Reply:
That's the next line nested under `fn` (data and options), this is how I've seen it documented otherwise.